doc/README/Design/Hierarchies.agda

```diff
@@ -66,7 +66,7 @@ private
 
 -- The Core module contains the basic units of the hierarchy.
 
--- For example for binary relations these are homoegeneous and
+-- For example, in binary relations these are homogeneous and
```

Comment:
Comma seems worthwhile, but not sure about the increase in legibility/intelligibility arising from the change in/of preposition, maybe going all out and writing
```suggestion
-- For example, in the case of binary relations these are homogeneous and
```
(possibly with knock-on consequences for the line-breaking/layout)